frontend/src/pages/modelServing/screens/projects/ServingRuntimeModal/ServingRuntimeTemplateSection.tsx

```diff
@@ -76,34 +107,252 @@ const ServingRuntimeTemplateSection: React.FC<ServingRuntimeTemplateSectionProps
     ),
   }));
 
+  const getServingRuntime = () => (
+    <>
+      <MenuGroup
+        data-testid="project-scoped-serving-runtimes"
+        label={
+          <Flex
+            spaceItems={{ default: 'spaceItemsXs' }}
+            alignItems={{ default: 'alignItemsCenter' }}
+            style={{ paddingBottom: '5px' }}
+          >
+            <FlexItem style={{ display: 'flex', paddingLeft: '12px' }}>
+              <img
+                style={{ height: 20, paddingTop: '3px' }}
```

Comment:
again for consistency i think `'20px'`

Reply:
also with this in 3 places, perhaps it could be a component? although it's small so maybe it doesn't matter